app/api/router/tokens.py

```diff
@@ -1,13 +1,14 @@
 from fastapi import APIRouter, HTTPException
 import requests
 
+from typing import List
 from schemas.TokenBalance import TokenBalance
 
 router = APIRouter()
 
 
 @router.get("/tokens/{chain}/{address}")
-async def tokens(chain: int, address: str) -> [TokenBalance]:
+async def tokens(chain: int, address: str) -> List[TokenBalance]:
```

Comment:
Ohh we need this for `Docs`? - `List`?

Reply:
pylint suggestion. It improves code readability.